src/main/kotlin/org/rust/lang/core/types/RustTypeFingerprint.kt

```diff
@@ -16,12 +18,14 @@ data class RustTypeFingerprint private constructor(
         fun create(type: RsTypeReference): RustTypeFingerprint? = when (type) {
             is RsBaseType -> type.path?.referenceName?.let(::RustTypeFingerprint)
             is RsRefLikeType -> type.typeReference?.let { create(it) }
+            is RsArrayTypeImpl -> RustTypeFingerprint("[T]")
             else -> null
         }
 
         fun create(type: RustType): RustTypeFingerprint? = when (type) {
             is RustStructType -> type.item.name?.let(::RustTypeFingerprint)
             is RustEnumType -> type.item.name?.let(::RustTypeFingerprint)
+            is RustSliceType -> RustTypeFingerprint("[T]")
```

Comment:
Hm, I am a bit worried (perf-wise) that we hash all array types to a single string. But perhaps this is OK? We do this for all other generics anyway.